src/Altinn.Platform/Altinn.Platform.Events/Events/Models/EventsSubscription.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace Altinn.Platform.Events.Models
+{
+    /// <summary>
+    /// Class that describes a events subscriptions
+    /// </summary>
+    public class EventsSubscription
+    {
+        /// <summary>
+        /// Endpoint to receive matching events
+        /// </summary>
+        public string EndPoint { get; set; }
```

Comment:
Any reason this is not a URI so we can have some help in verifying it's validity?

Reply:
Lets try. I am not 100% sure if it could be any serilazation issues